src/Config/Config.py

```diff
@@ -88,3 +92,30 @@ def remove_proxy(self):
             config_file.seek(0)
             config_file.truncate()
             config_file.write(json.dumps(config))
+
+    def set_user(self, user):
+        api_url = "https://codeforces.com/api/user.info?handles="
+        response = requests.get(url = api_url + user, proxies = self.get_proxy())
+        if(response.status_code != 200):
+            raise UsernameNotFound(user)
+        html_content = response.json()
+        config_file_path = self.config_file_path
+        with open(config_file_path, 'r+') as config_file:
+            config = json.load(config_file)
+            config["user"]["firstname"] = html_content["result"][0]["firstName"]
+            config["user"]["lastname"] = html_content["result"][0]["lastName"]
+            config["user"]["rating"] = str(html_content["result"][0]["rating"]),
+            config["user"]["contri"] = str(html_content["result"][0]["contribution"]),
+            config["user"]["rank"] = str(html_content["result"][0]["rank"]),
+            config["user"]["maxrating"] = str(html_content["result"][0]["maxRating"])
```

Comment:
There's an error over here which is causing creation of an list. Try to figure it out.

Reply:
The commas :cry: 
fixed it